Beam/resources/assets/js/remplib.js

```diff
@@ -657,7 +662,7 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
             }
 
 
-            const params = {
+            var params = {
                 "action": "progress",
```

Comment:
@rootpd Pri testovani mi backend vratil chybu ze action moze byt len "load" alebo "timespent". Je to vec ktoru my riesit nemusime ?